lib/broken_record/slack_aggregator.rb

```diff
@@ -8,12 +9,33 @@ def report_final_results
         username: "#{app_name} ValidationMaster"
       })
 
+      send_summary(notifier)
+      send_snippet(notifier)
+    end
+
+    private
+
+    def send_summary(notifier)
       if success?
         notifier.send!("\nAll models validated successfully.")
       else
         notifier.send!("\n#{@total_errors} errors were found while running validations.")
       end
     end
 
+    def send_snippet(notifier)
+      # Our snippet should naively show the output from the ConsoleAggregator
+      console_aggregator = BrokenRecord::ConsoleAggregator.new
+      all_results.each{ |result| console_aggregator.add_result(result) }
+      # Store the output of the console aggregator into a StringIO object
+      $stdout = StringIO.new
```

Comment:
@AlexEvanczuk I'm assuming you tested this manually, I'm not sure how the nippit should work, but seems like you override snippet here on line 33 from line 32 and just want to make sure that is intentional!